bindings/core/src/method/wallet.rs

```diff
@@ -332,6 +344,25 @@ pub enum WalletMethod {
         delegation_id: DelegationId,
         reclaim_excess: bool,
     },
+    /// Prepare to begin staking.
+    /// Expected response: [`PreparedTransaction`](crate::Response::PreparedTransaction)
+    PrepareBeginStaking {
+        params: BeginStakingParams,
+        #[serde(default)]
+        options: Option<TransactionOptions>,
+    },
+    #[serde(rename_all = "camelCase")]
```

Comment:
No `Expected response` comments for the others :( ?

Reply:
oop